teleop_twist_keyboard.py

```diff
@@ -19,18 +19,30 @@
 e/c : increase/decrease only angular speed by 10%
 anything else : stop
 
+For Horonomic base, holding down Shift key
+
+For Drones, use t/b to control elevation
```

Comment:
These instructions are pretty vague; they still leave open questions about which keys are active when holding shift, and how `t` and `b` control elevation.